exercises/scrabble-score/instructions.md

```diff
@@ -0,0 +1,29 @@
+# Instructions
+
+Compute a word's Scrabble score by summing its letters' values.
```

Comment:
```suggestion
Your task is to compute a word's Scrabble score by summing the values of its letters.
```

I suggest this slight variation so we don't have the same phrase "summing it's letter's values" twice in a row.